scripts/load_safe_deployments.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import requests
+
+safe_deployments_base_url = (
+    "https://raw.githubusercontent.com/safe-global/safe-deployments/main/src/assets"
+)
+safe_deployments_path = "../gnosis/safe/safe-deployments/safe-deployments.json"
+
+deployments = [
```

Comment:
Don't hardcode all this. Open the directory (maybe you need to clone it) and check the files inside, using something like `os.walk()`. That way we don't have to adapt the script if there's a new release or a new file is added